src/components/reduce-item-by-one.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useShoppingCart, formatCurrencyString } from 'use-shopping-cart'
+
+export function ReduceItemByOne({ product }) {
+  const { reduceItemByOne } = useShoppingCart()
+
+  /* A helper function that turns the price into a readable format */
+  const price = formatCurrencyString({
+    value: product.price,
+    currency: product.currency,
+    language: 'en-US',
```

Comment:
For i18n, do we want `'en-US'` or do we want `navigator.language` here?

Reply:
@ChrisBrownie55 Because these docs are made with Gatsby, during build time, this fails. I'm going to make another pull request to get navigator working, gotta do some stuff to account for browser not being available. This was just a duct tape solution so I can get it live